packages/client/hmi-client/src/components/workflow/ops/model-from-equations/tera-model-from-equations-drilldown.vue

```diff
@@ -38,7 +38,7 @@
 					</header>
 					<Accordion :active-index="0">
 						<AccordionTab header="Model equations">
-							<h6 class="pb-2">Use these equation</h6>
+							<h6 class="pb-2">Use these equations</h6>
```

Comment:
If you want to be pedantic about it...
```suggestion
							<h6 class="pb-2">Use {{ includedEquations.length > 1 ? 'these equations' : 'this equation' }}</h6>
```